docs/timeseries/index.rst

```diff
@@ -48,12 +48,10 @@ source::
 
 .. note::
     The light curve provided here is handpicked for example purposes. For
-    more information about the Kepler FITS format, see the `Kepler Data
-    Validation Document
-    <https://exoplanetarchive.ipac.caltech.edu/docs/KeplerDV.html>`_ and the
-    Kepler Science Center `Light Curve Files
-    <https://keplerscience.arc.nasa.gov/PyKEprimerLCs.shtml>`_ documentation. To
-    get other Kepler light curves for science purposes using Python, see the
+    more information about the Kepler FITS format, see
+    `Section 2.3.1 of the Kepler Archive Manual
+    <https://archive.stsci.edu/kepler/manuals/archive_manual.pdf#page=16>`_.
```

Comment:
Does it have to be a PDF?

Reply:
An alternative link would be [this notebook](https://github.com/spacetelescope/notebooks/blob/master/notebooks/MAST/Kepler/Kepler_Lightcurve/kepler_lightcurve.ipynb), but it's not as authoritative as the Kepler manual.  Would it help if I added `(pdf)` after the link?

Reply:
No, link check worked, so good enough for me. Thanks!